include/LVSS/LVSS.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef _LVSS_
+#define _LVSS_
+
+#pragma once
```

Comment:
You need either the header guard or `#pragma once`, but you don't need both

Reply:
ok dude